src/main/java/uk/gov/hmcts/reform/idam/notify/client/NotifyService.java

```diff
@@ -57,18 +58,24 @@ public void sendEmail(String from, String to, String subject, String message, St
 
         final String languageSpecificNotificationClientTemplateId;
 
-        LOGGER.fine(String.format("Sending OTP to: %s, subject: %s", to, subject));
-
         // We are hijacking the i18n mechanism here. The language-specific templateId is being passed as message subject.
         // This condition makes it backwards-compatible.
+        // This path is for IDAM >=2.1
         if (OPENAM_DEFAULT_SUBJECT.equalsIgnoreCase(subject)) {
-            LOGGER.warning("Language-specific Notify templateId is not available. Falling back to the one supplied in env.");
-            languageSpecificNotificationClientTemplateId = notificationClientTemplateId;
+            // This path is for IDAM >=2.2 which uses AuthTrees and requires a different language injection mechanism
+            // This time using the "from" filed.
+            if (OPENAM_DEFAULT_SENDER.equalsIgnoreCase(to)) {
```

Comment:
I'm guessing that you intended to have `OPENAM_DEFAULT_SENDER.equalsIgnoreCase(from)` here not `to`.

Reply:
Yes, good spot.